app/views/admin/republishing/confirm.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :page_title, "Are you sure you want to republish \"#{@document_title}\"?" %>
+<% content_for :title, "Are you sure you want to republish \"#{@document_title}\"?" %>
+<% content_for :title_margin_bottom, 6 %>
+<div class="govuk-grid-row">
+  <section class="govuk-grid-column-two-thirds">
+    <p class="govuk-body govuk-!-margin-bottom-7">
+      This will schedule the page to be republished.
+    </p>
+
+    <%= button_to("Confirm republishing",
```

Comment:
"Confirm republish request"?

Reply:
I think I prefer the original, unless you've got strong feelings about this one?